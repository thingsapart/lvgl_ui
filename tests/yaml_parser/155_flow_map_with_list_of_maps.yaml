# Tests a more complex flow-style map.
{
  title: "My List",
  items: [{id: 1, type: 'A'}, {id: 2, type: 'B'}]
}
