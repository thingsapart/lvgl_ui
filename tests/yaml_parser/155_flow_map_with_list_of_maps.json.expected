[{
		"title":	"My List",
		"items":	[{
				"id":	1,
				"type":	"A"
			}, {
				"id":	2,
				"type":	"B"
			}]
	}]
